packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web5/api",
-  "version": "0.8.3",
+  "version": "0.8.2",
```

Comment:
```suggestion
  "version": "0.8.4",
```

right?

Reply:
no bump down to `0.8.2` to avoid triggering a premature release of that package until Dev Programs completes their testing and the dev community has had a couple days notice that the hosted DWN servers will be bounced.

Reply:
Oh my b, I see you commented on that earlier, I jumped right into code diff view